config/initializers/ios_apn.rb

```diff
@@ -0,0 +1,9 @@
+if Rails.env.production?
+  IOS_APN = ENV['IOS_APN'] || Houston::Client.production
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.